src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import { Todo } from '../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const TodoList: React.FC<{ todos: Todo[] }> = ({ todos }) => (
+  <>
+    <ul className="todo__list">
+      {todos.map(todo => (
+        <li className="todo__item">
+          <UserInfo {...todo} />
+          <TodoInfo {...todo} />
```

Comment:
не логічно що userInfo отримує якусь тудушку. Якщо ти захочеш перевикористати де інде де немає тудушки?у
```suggestion
          <UserInfo user={user} />
          <TodoInfo todo={todo} />
```